pom.xml

```diff
@@ -5,19 +5,25 @@
     <parent>
         <groupId>life.qbic</groupId>
         <artifactId>parent-pom</artifactId>
-        <version>2.4.0-SNAPSHOT</version>
+        <version>3.0.0</version>
     </parent>
-
     <artifactId>xml-manager-lib</artifactId>
     <packaging>jar</packaging>
     <version>1.6.0-SNAPSHOT</version>
     <name>XML Property Parser Library</name>
     <url>http://maven.apache.org</url>
     <dependencies>
     		<dependency>
-			<groupId>org.apache.commons</groupId>
-			<artifactId>commons-lang3</artifactId>
-		</dependency>
+          <groupId>org.apache.commons</groupId>
+          <artifactId>commons-lang3</artifactId>
+            <scope>${osgi.scope}</scope>
+		    </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-core</artifactId>
+            <version>2.2.7</version>
+            <scope>${osgi.scope}</scope>
+        </dependency>
```

Comment:
where does this come from?

Reply:
The sun.xml.bind package has to be added explicitly since it's not provided by JDK 1.11. 
Previously we got it for "free" because we used JDK 1.8.